docs/our_work/linkage.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Data Linkage Enhancement'
+summary: 'The data linkage enhancement project encompasses all things data linkage, from documenting the existing state of linkage in NHS England in the MPS handbook, to exploring better matching algorithms using Splink, to creating a quality assurance framework for data linkage.'
+origin: 'NHS Digital'
+tags: ['BEST PRACTICE', 'LINKAGE', 'PYTHON', 'QUALITY']
+---
+![Diagram representing the three current areas of the data linkage project: Quality Assurance, Better Matching Algorithm, and Data Linkage as a service.](../images/linkage_visual_abstract.png)
+
+Data linkage is an essential part of being able to use data within any organisation, but in particular the NHS. Being able to trace patients across their care journey, making underrepresented populations not be lost in the cracks, and ensuring compatibility when using several data sets for the same purpose is a pillar of why the data linkage team exists, and why we do what we do. 
+
+# **Work we do**
+
+## Data Linkage as a service
```

Comment:
Order these as they are in the diagram - for me, the Service comes last, as you've got better algorithms, you've set up a quality assurance framework - but you're not just leaving people to figure it out themselves - you're also setting up a service to actually help them.